amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/dashboard/model/ConsumerInfo.java

```diff
@@ -48,4 +48,13 @@ public long getNextSnapshotId() {
   public void setNextSnapshotId(long nextSnapshotId) {
     this.nextSnapshotId = nextSnapshotId;
   }
+
+  public AmoroSnapshotsOfTable getAmoroPreviousSnapshotsOfTable() {
```

Comment:
I don’t quite understand why the return the previous snapshot information . Can you give some examples? Why is the current snapshot information not displayed, but the previous one?

Reply:
I misnamed the description here. nextSnapshotId-1 is the current Snapshot consumption information